QA/manual_testing.md

```
@@ -368,53 +496,81 @@
-| 4.2        |          |            |      |      |
-| 4.3        |          |            |      |      |
-| 4.4        |          |            |      |      |
-| 5.1        |          |            |      |      |
-| 5.2        |          |            |      |      |
-| 5.3        |          |            |      |      |
-| 6.1        |          |            |      |      |
-| 6.2        |          |            |      |      |
-| 6.3        |          |            |      |      |
\ No newline at end of file
+| ...        |          |            |      |      |
+| 8.2        |          |            |      |      |
```

markdownlint-cli2 (0.17.2)

Files should end with a single newline character
null

(MD047, single-trailing-newline)